ext/bcmath/tests/negative_scale.phpt

```diff
@@ -0,0 +1,70 @@
+--TEST--
```

Comment:
This test fails with a [memory leak report](https://travis-ci.org/github/php/php-src/jobs/679437712#L2325). Please have look.

Reply:
The memory leak is likely caused by bailing out of `bcpowmod()`. You can probably resolve that by moving the checking code directly after the ZPP.

Reply:
done. By the way, how do you debug tests that "works on my machine"?

Reply:
In this case `configure --enable-debug …` should have shown the issue. Better [memory issue checking/debugging](http://www.phpinternalsbook.com/php7/memory_management/memory_debugging.html?highlight=valgrind) can be done valgrind.

Reply:
the `configure --enable-debug` already was enabled. 

Maybe I'm doing something wrong, but valgrind also didn't show any errors
the command that I used
```bash
ZEND_DONT_UNLOAD_MODULES=1 USE_ZEND_ALLOC=0 valgrind --leak-check=full --show-reachable=yes --track-origins=yes /usr/local/bin/php run-tests.php -j6 ext/bcmath/

```

Reply:
This way you apply valgrind to run-tests.php, but not to the actual tests. Use `/usr/local/bin/php run-tests.php -m -j6 ext/bcmath/` (or `make test TESTS="-m -j6 ext/bcmath"`) instead.